torch/_higher_order_ops/hints_wrap.py

```diff
@@ -0,0 +1,120 @@
+# mypy: allow-untyped-defs
```

Comment:
We should import torch/_higher_order_ops/hint_wraps.py in _higher_order_ops/__init__.py

Reply:
Do you mean the usage of `from torch._higher_order_ops import hints_wrapper` rather than `from torch._higher_order_ops.hints_wrapper import hints_wrapper`?